src/main/java/de/bonndan/nivio/model/RelationType.java

```diff
@@ -31,7 +32,18 @@ public static RelationType from(String relationType) {
      * @return filtered subset
      */
     @Deprecated
-    public List<RelationItem> filter(Collection<? extends RelationItem> all) {
-        return all.stream().filter(relationItem -> this.equals(relationItem.getType())).collect(Collectors.toList());
+    public List<Relation> filter(Collection<? extends Relation> all) {
+        return all.stream().filter(relation -> this.equals(relation.getType())).collect(Collectors.toList());
+    }
+
+    /**
+     * Filters a list of relations for equal type.
+     *
+     * @param all all relations
+     * @return filtered subset
+     */
+    @Deprecated
+    public List<RelationDescription> filterRelationDescription(Collection<? extends RelationDescription> all) {
+        return all.stream().filter(relation -> this.equals(relation.getType())).collect(Collectors.toList());
```

Comment:
This is another duplication. These are only used in tests. Do we still need those? What are we testing with those that our code is not doing? Why are they deprecated?

Reply:
In earlier versions the relation type had an effect on layouting, but that has been removed. Relation type filtering can be done using RelationType.equals(), so these methods can be removed.

Reply:
I will do that.